tox.ini

```diff
@@ -1,9 +1,11 @@
 [tox]
-envlist = flake8-py3, py27, py34, py35, py36, py37, py38, pypy
+envlist = flake8-py3, py27, py34, py35, py36, py37, py38, pypy, py{27,38}-nobrotli
```

Comment:
Wish there was a better way to handle this.

Reply:
I am not very good with tox, and open to constructive criticism here.